### DISCUSSION TITLE
What the question marks `?` means?

### DISCUSSION THREAD

**Discussion by jlenriquez** [basecamp/omakub#341] [Q&A] [answered]
Hello,

It may not be related to omakub but with neovim the way it is configured, etc. and this might be a silly question but I would like to understand what the `?` marks means here?:
<img width="290" alt="questionmarks" src="https://github.com/user-attachments/assets/149a536e-f02b-454e-9375-d427e1d1971f">

I saw some omakub demos in youtube having the same, but a bit curious about it.

Thank you!

**Comment by atenghamn** [comment, marked as answer]
I think thats because you haven't added them to your repo
https://www.reddit.com/r/neovim/comments/12hexgy/question_marks_in_almost_every_file_of_a/?rdt=62078
